bin/drebuild

```diff
@@ -1,18 +1,29 @@
 #!/bin/sh
 
 echo "Remove all unused images before we build new ones…"
+echo "\n\033[36m===> docker image prune -f\033[0m\n"
 docker image prune -f
 
 echo "Rebuilding the test server…"
+echo "\n\033[36m===> docker-compose --file=docker-compose.test.yml down -v --remove-orphans\033[0m\n"
 docker-compose --file=docker-compose.test.yml down -v --remove-orphans
+echo "\n\033[36m===> docker-compose --file=docker-compose.test.yml build\033[0m\n"
```

Comment:
As we are now using `bin/dtest-server` I think this command is being run twice, I noticed as I was getting the output from entrypoint.sh twice in the output.